### ISSUE TITLE
provider/aws: aws_vpc operations fail in v0.6.10

### ISSUE THREAD

**Issue by laggyluke** [hashicorp/terraform#4874]
Possibly related to #3994, might be due to `eu-central-1` region not supporting VPC ClassicLink or something like that.

Configuration:
```
provider "aws" {
    region = "eu-central-1"
}

resource "aws_vpc" "test" {
    cidr_block = "10.0.0.0/16"
}
```

Plan:
```
$ terraform plan
Refreshing Terraform state prior to plan...

[skipped]

+ aws_vpc.test
    cidr_block:                "" => "10.0.0.0/16"
    default_network_acl_id:    "" => "<computed>"
    default_security_group_id: "" => "<computed>"
    dhcp_options_id:           "" => "<computed>"
    enable_classiclink:        "" => "0"
    enable_dns_hostnames:      "" => "<computed>"
    enable_dns_support:        "" => "<computed>"
    main_route_table_id:       "" => "<computed>"


Plan: 1 to add, 0 to change, 0 to destroy.
```

Apply:
```
$ terraform apply
aws_vpc.test: Creating...
  cidr_block:                "" => "10.0.0.0/16"
  default_network_acl_id:    "" => "<computed>"
  default_security_group_id: "" => "<computed>"
  dhcp_options_id:           "" => "<computed>"
  enable_classiclink:        "" => "0"
  enable_dns_hostnames:      "" => "<computed>"
  enable_dns_support:        "" => "<computed>"
  main_route_table_id:       "" => "<computed>"
Error applying plan:

1 error(s) occurred:

* aws_vpc.test: UnsupportedOperation: The functionality you requested is not available in this region.
	status code: 400, request id:

Terraform does not automatically rollback in the face of errors.
Instead, your Terraform state file has been partially updated with
any resources that successfully completed. Please address the error
above and apply again to incrementally change your infrastructure.
```

Refresh:
```
$ terraform refresh
aws_vpc.test: Refreshing state... (ID: vpc-a1bbd9c8)
Error refreshing state: 1 error(s) occurred:

* aws_vpc.test: UnsupportedOperation: The functionality you requested is not available in this region.
	status code: 400, request id:
```

The configuration above works fine on v0.6.9.

**Comment by phinze** [comment]
Thanks for the report, @laggyluke. We're investigating this now.

**Comment by catsby** [comment]
https://github.com/hashicorp/terraform/pull/4879 should fix this, we're double checking things, thanks!

**Comment by catsby** [comment]
We merged #4897, thanks for the report! We're going to close this for now, let us know if you need anything else.

**Comment by laggyluke** [comment]
Thanks a lot for your work!
